programs/cleos/main.cpp

```diff
@@ -1057,16 +1057,16 @@ struct set_action_permission_subcommand {
       auto permissions = actionRoot->add_subcommand("permission", localized("Set parameters dealing with account permissions"));
       permissions->add_option("account", accountStr, localized("The account to set/delete a permission authority for"))->required();
       permissions->add_option("code", codeStr, localized("The account that owns the code for the action"))->required();
-      permissions->add_option("type", typeStr, localized("The type of the action"))->required();
+      permissions->add_option("type", typeStr, localized("The type of the action [use ALL for all types]"))->required();
       permissions->add_option("requirement", requirementStr, localized("[delete] NULL, [set/update] The permission name require for executing the given action"))->required();
```

Comment:
I think it may be worth considering updating the text of this item to be similar to the formatting and syntax above. Something like
> The permission name required for executing the given action [use NULL to delete]

or something